README.md

```diff
@@ -19,34 +19,38 @@ yarn install
 ```
 
 <!-- AUTO-GENERATED-CONTENT:START (PACKAGES:packagePath=packages/manager/modules&name=modules&level=2) -->
+
 ## Availables modules
 
-| Package | Version | Dependencies | Dev dependencies | Peer dependencies | Changelog |
-| ------- | ------- | ------------ | ---------------- | ----------------- | :-------: |
-| [@ovh-ux/manager-cloud-styles](https://github.com/ovh-ux/manager/tree/master/packages/manager/modules/cloud-styles) | [![npm version](https://badgen.net/npm/v/@ovh-ux/manager-cloud-styles)](https://www.npmjs.com/package/@ovh-ux/manager-cloud-styles) | [![Dependencies](https://badgen.net/david/dep/ovh-ux/manager/packages/manager/modules/cloud-styles)](https://npmjs.com/package/@ovh-ux/manager-cloud-styles?activeTab=dependencies) | [![Dev Dependencies](https://badgen.net/david/dev/ovh-ux/manager/packages/manager/modules/cloud-styles)](https://npmjs.com/package/@ovh-ux/manager-cloud-styles?activeTab=dependencies) | [![Peer Dependencies](https://badgen.net/david/peer/ovh-ux/manager/packages/manager/modules/cloud-styles)](https://npmjs.com/package/@ovh-ux/manager-cloud-styles?activeTab=dependencies) | [:books:](https://github.com/ovh-ux/manager/blob/master/packages/manager/modules/cloud-styles/CHANGELOG.md) |
+| Package                                                                                                                                      | Version                                                                                                                                                     | Dependencies                                                                                                                                                                                                 | Dev dependencies                                                                                                                                                                                                 | Peer dependencies                                                                                                                                                                                                  |                                                         Changelog                                                        |
```

Comment:
If you want update the main `README.md` file, you could run `yarn run docs:update-readme` and you will see your module and app appears in lists.

Reply:
Readme has been updated using yarn run docs:update-readme command. Thanks for pointing this out